### PR TITLE
Baremetal Plugin API docs added

### PR DESCRIPTION
**Pull request by shehzabcodecraft**

### CODE REVIEW

File: source/includes/baremetal/devices.md [hunk 1]
```diff
@@ -0,0 +1,690 @@
...
+| `deviceDetail.deviceIPDetail.primaryIp` <br/>_string_        |                                                                                                                                          |
+| `deviceDetail.deviceIPDetail.description` <br/>_string_      |                                                                                                                                          |
+| `deviceDetail.deviceIPDetail.gatewayIp` <br/>_string_        |                                                                                                                                          |
+| `deviceDetail.deviceIPDetail.subnetMask` <br/>_string_       |                                                                                                                                          |
+| `deviceDetail.deviceIPDetail.usableIps` <br/>_Array[string]_ |                                                                                                                                          |
```
Comment (fyrnaga) [inline source/includes/baremetal/devices.md:197]: There is no description for these fields.

File: source/includes/baremetal/devices.md [hunk 2]
```diff
@@ -0,0 +1,690 @@
...
+| `deviceInitialPassword.passwordExpires` <br/>_string_        | Number of days password remains / Expired                                                                                                |
+| `deviceInitialPassword.port` <br/>_int_                      | Port for initial ssh access.                                                                                                             |
+| `deviceInitialPassword.user` <br/>_int_                      | User for initial ssh access.                                                                                                             |
+| `deviceIPs.subnet` <br/>_string_                             |                                                                                                                                          |
+| `deviceIPs.netmask` <br/>_string_                            |                                                                                                                                          |
```
Comment (fyrnaga) [inline source/includes/baremetal/devices.md:205]: No description for these fields

**Review by fyrnaga — APPROVED** [review]

File: source/includes/baremetal/devices.md [hunk 3]
```diff
@@ -0,0 +1,690 @@
...
+| `id`<br/>_string_                      | The unique ID of the device.                                                                 |
+| `servicePlan`<br/>_string_             | The unique ID of the service associated with this device.                                    |
+| `name`<br/>_string_                    | User given custom name.                                                                      |
+| `deviceType` <br/>_string_             | Generic description of device. Usually type and rack unit size.                              |
+| `primaryIp` <br/>_string_              | The first assigned public IP for accessing this device.                                      |
+| `status` <br/>_string_                 | active/inactive                                                                              |
+| `monitorsTotal`<br>_int_               | Total # device monitors                                                                      |
+| `monitorsUp`<br/>_int_                 | Of passing device monitors.                                                                  |
```
Comment (patrickspensieri) [inline source/includes/baremetal/devices.md:78]: Should be "Number of ...", same for other APIs

File: source/includes/baremetal/devices.md [hunk 4]
```diff
@@ -0,0 +1,690 @@
...
+| `ipmiAddress` <br/>_string_            | IP address for IPMI connection. Requires you to whitelist your current IP or be on IPMI VPN. |
+| `powerStatus` <br/>_string_            | ON/OFF                                                                                       |
+| `tags`<br/>_Array[string]_             | List of all user set device tags.                                                            |
+| `hostname` <br/>_string_               | A FQDN for the device. for example: example.hivelocity.net.                                  |
```
Comment (patrickspensieri) [inline source/includes/baremetal/devices.md:82]: Should be "For example" with a capital f

File: source/includes/baremetal/devices.md [hunk 5]
```diff
@@ -0,0 +1,690 @@
...
+| `primaryIp` <br/>_string_                                    | The first assigned public IP for accessing this device.                                                                                  |
+| `status` <br/>_string_                                       | active/inactive                                                                                                                          |
+| `monitorsTotal`<br>_int_                                     | Total # device monitors                                                                                                                  |
+| `monitorsUp`<br/>_int_                                       | Of passing device monitors.                                                                                                              |
+| `monitors`<br/>_string_                                      | monitors.                                                                                                                                |
```
Comment (patrickspensieri) [inline source/includes/baremetal/devices.md:179]: Could this description be improved?

File: source/includes/baremetal/sshkey.md [hunk 6]
```diff
@@ -0,0 +1,121 @@
+## SSH Keys
+
+Allows you to create, delete, and manage your SSH keys.
+
+Create and manage your SSH keys.
```
Comment (patrickspensieri) [inline source/includes/baremetal/sshkey.md:5]: Just one of these descriptions is enough

**Review by patrickspensieri — COMMENTED** [review]
Left a few minor comments

File: source/includes/baremetal/devices.md (hunk 3 above)
Comment (shehzabcodecraft) [reply source/includes/baremetal/devices.md:78]: Resolved.

File: source/includes/baremetal/devices.md (hunk 4 above)
Comment (shehzabcodecraft) [reply source/includes/baremetal/devices.md:82]: Resolved

File: source/includes/baremetal/sshkey.md (hunk 6 above)
Comment (shehzabcodecraft) [reply source/includes/baremetal/sshkey.md:5]: Resolved

File: source/includes/baremetal/devices.md (hunk 1 above)
Comment (shehzabcodecraft) [reply source/includes/baremetal/devices.md:197]: We don't have descriptions for these fields from the vendor.

File: source/includes/baremetal/devices.md (hunk 5 above)
Comment (shehzabcodecraft) [reply source/includes/baremetal/devices.md:179]: We don't have descriptions for these fields from the vendor.

**Review by patrickspensieri — APPROVED** [review]

**Pull request merged**